src/frontend/components/header/header.ts

```diff
@@ -29,31 +27,28 @@ type Route = any; // TODO(cbond): use real Route type
   host: {
     '(document:click)': 'resetIfSettingOpened($event)'
   },
-  directives: [
-    Search,
-  ]
 })
 export class Header {
   @ViewChild('theme-menu') menuElement: ElementRef;
   @ViewChild('theme-menu-button') menuButtonElement: ElementRef;
 
-  @Input() private selectedTab: Tab;
-  @Input() private options: Options;
-  @Input() private tree: MutableTree;
-  @Input() private routerTree: Array<Route>;
+  @Input() selectedTab: Tab;
```

Comment:
@xorgy what's the reason for getting rid of private properties and methods in this class?